src/dune_rpc_impl/dune_rpc_impl.ml

```diff
@@ -5,6 +5,7 @@ module For_handlers = For_handlers
 module Private = Dune_rpc_client.Private
 module Watch_mode_config = Watch_mode_config
 module Where = Dune_rpc_client.Where
+module Diagnostics = Diagnostics
```

Comment:
Since you only need one function, you can just export the function you need. Also, it should be accessible as `Diagnostics.For_tests.xxx`.